tremor-cli/tests/integration/tcp-reuseport/config.troy

```diff
@@ -10,7 +10,7 @@ flow
       "separate"
     ],
     config = {
-      "url": "127.0.0.1:22224",
+      "default_handle": "a",
```

Comment:
anotehr not for later, we should see if we can avoid a default handle config it seems to make things complicated and error prone. I'm not sure what a better approach is would it's something to think about as this seems wrong. Perhaps the "first": handle can be the default?